components/clarinet-deno/ext/stacksjs-helper-generator.ts

```diff
@@ -127,62 +124,42 @@ Clarinet.run({
           if (f.args.length > 0) {
             // Generate code for interface
             code.push([
-              `            export interface ${
-                kebabToCamel(capitalize(f.name))
-              }Args {`,
+              `            export interface ${kebabToCamel(
+                capitalize(f.name)
+              )}Args {`,
             ]);
-            for (let arg of f.args) {
+            for (const arg of f.args) {
               code.push([
-                `                ${kebabToCamel(arg.name)}: ${
-                  typeToCVType(arg.type)
-                },`,
+                `                ${kebabToCamel(arg.name)}: ${typeToCVType(
+                  arg.type
+                )},`,
               ]);
             }
-            code.push([
-              `            }`,
-              ``,
-            ]);
+            code.push([`            }`, ``]);
 
             // Generate code for helper function
             code.push([
-              `            export function args(args: ${
-                kebabToCamel(capitalize(f.name))
-              }Args): ClarityValue[] {`,
+              `            export function args(args: ${kebabToCamel(
+                capitalize(f.name)
+              )}Args): ClarityValue[] {`,
               `                return [`,
             ]);
-            for (let arg of f.args) {
+            for (const arg of f.args) {
               code.push([
                 `                    args.${kebabToCamel(arg.name)},`,
               ]);
             }
-            code.push([
-              `                ];`,
-              `            }`,
-              ``,
-            ]);
+            code.push([`                ];`, `            }`, ``]);
           }
 
-          code.push([
-            `        }`,
-            ``,
-          ]);
+          code.push([`        }`, ``]);
         }
 
-        code.push([
-          `    }`,
-        ]);
+        code.push([`    }`]);
       }
 
-      code.push([
-        `}`,
-        ``,
-      ]);
+      code.push([`}`, ``]);
     }
-
-    const write = await Deno.writeTextFile(
-      "./artifacts/contracts.ts",
-      code.flat().join("\n"),
-    );
   },
 });
```

Comment:
Nice, thanks. could you please run this extension on a contract, and make sure that the output is correctly formatted?